UI/Console.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Console.h"
+
+
+std::vector<std::string> logs;
```

Comment:
Nitpick: I generally don't like declaring variables inside the cpp files because it can get lost amidst function definitions.

Is there something preventing variables like this from being in the header files?

Reply:
Putting in the header gives an error about being initialized multiple times and I am unable to figure out how to fix, it works fine if put in cpp